applications/vns-domains.md

```diff
@@ -0,0 +1,175 @@
+- Use MerkleTree for whitelist
+- Store metadata on-chain
+- No oracle, uses a stablecoin called $VeUSD*
+- No exponential premium pricing, prices are static
+- Minimum registration for domain is 365 days (For ENS, it’s 28 days)
+- No DNS-related functionalities
+
+*Subject to change based on $VeUSD availability
+
+#### Documentation
+VNS Domains has a provided documentation consisting of frequently asked questions, policies and terminology. This documentation will be regularly updated as needed.
+* https://www.vns.domains/#faq (FAQ)
+* https://docs.vns.domains/ (Documentation Site)
+
+### Ecosystem Fit
+
+VeChain Name Service (VNS) is a decentralized naming service similar to Ethereum Naming Service (ENS) and NF Domains on the Algorand blockchain. VNS, like these other services, aims to improve the usability of blockchain interactions by allowing users to customize their VeChain wallet address with a human-readable format. The critical difference is that VNS is the only naming service that will soon be available on the VeChainThor blockchain, making it a one-of-a-kind and necessary project for the VeChain ecosystem. In addition, VNS is set to integrate with VeChain wallets, marketplaces, and other DApps, providing an easy-to-use naming convention for on-chain transactions, which will help drive the VeChain ecosystem's adoption and growth.
+
+## Team 
+
+The VeChain Name Service (VNS) team is a small but highly skilled and experienced group of developers and business professionals. They have a deep understanding of the VeChainThor blockchain and are passionate about building innovative solutions to help grow the VeChain ecosystem. The VNS team is confident that they have the technical skills and experience needed to complete the task of building a decentralized name service for the VeChainThor blockchain. They are committed to providing a user-friendly and secure platform that will allow VeChain users to customize their wallet addresses and improve the usability of interactions within the blockchain.
+
+It should be noted that VNS domains was a project originally started by another enitity not affiliated with the current team. The only assets that were transitioned to the current team was the name "VNS Domains" and the domain name www.vns.domains. No previous financial grants were award to the current team by Vechain foundation or any other entity.
+
+### Team members
+
+Kerryn Pegram (Kiwi) - Co-founder, CEO
+
+Virgo - Co-founder, Writes code but sometimes makes art
+
+idr0p - Co-founder, Business Development
+
+### Team Website
+
+- https://vns.domains
+
+### Team's experience
+
+The VNS team has decades of experience in business and software engineering, serving industries ranging from Insurance to IT.
+
+### Team Code Repos
+
+Available upon request.
+
+### Team LinkedIn Profiles
+
+Available upon request.
+
+## VNS Development Roadmap 
+
+#### Overview
+
+|  | Current Status| Milestone 1 | Milestone 2 | Milestone 3 |
```

Comment:
Please remove `Current Status` and add `Total`. And also keep in mind that we have a [rule](https://github.com/vechain/grant-program/blob/master/applications/application-template.md#overview-1) that limits the milestone 1 funds cannot exceed 40% of the total amount you are applying for.